kubernetes/keycloak/theme-provider/themes/hmda/login/resources/js/register.js

```diff
@@ -117,7 +117,7 @@
                   "<span>Sorry, we couldn't find that email domain. For help getting registered, please fill out and submit </span>"
                 ),
                 getHelpFormLink(),
-                $('<span>.</span>')
+                $("<span>.</span> You must use a financial institution's domain to register, personal email domains are not accepted.")
```

Comment:
@meissadia  would this comment need to be within `<span></span>`?

Reply:
@PatrickGoRaft Excellent catch, thank you.  Updated.